src/otx/algo/detection/utils/utils.py

```diff
@@ -424,6 +425,71 @@ def distance2bbox(
     return bboxes
 
 
+def distance2bbox_export(points: Tensor, distance: Tensor, max_shape: Tensor | None = None) -> Tensor:
```

Comment:
Can we add the argument `export=False` to `distance2bbox` and `clip_bboxes` for reusing the code blocks?

Reply:
Thanks for the recommendation.
I'll do it when refactoring duplicated functions.